core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -69,7 +69,10 @@ object ApiVersion {
     "0.11.0-IV1" -> KAFKA_0_11_0_IV1,
     // Introduced leader epoch fetches to the replica fetcher via KIP-101
     "0.11.0-IV2" -> KAFKA_0_11_0_IV2,
-    "0.11.0" -> KAFKA_0_11_0_IV2
+    "0.11.0" -> KAFKA_0_11_0_IV2,
+    // Introduced LeaderAndIsrRequest V1, UpdateMetadataRequest V4 and FetchRequest V6 via KIP-112
```

Comment:
I am not sure whether we should add comment regarding the ProduceRequest. @junrao said earlier that we only need to add comments regarding requested used between brokers. I guess we can do this in the other followup patch if needed.